doc/release_notes_1.1.rst

```diff
@@ -0,0 +1,519 @@
+
+-----
+
+:acrn-issue:`2267` - [APLUP2][LaaG] LaaG can't detect 4k monitor
+   After launching UOS on APL UP2 , 4k monitor cannot be detected.
+
+   **Impact:** UOS can't display on a 4k monitor.
+
+   **Workaround:** Use a monitor with less than 4k resolution.
+
+-----
+
+:acrn-issue:`2279` - [APLNUC] After exiting UOS, SOS can't use USB keyboard and mouse
+   After exiting UOS with mediator Usb_KeyBoard and Mouse, SOS cannot use the USB keyboard and mouse.
+
+   These steps reproduce the issue:
+
+   1) Insert USB keyboard and mouse in standard A port (USB3.0 port)
+   2) Boot UOS by sharing the USB keyboard and mouse in cmd line:
+
+      ``-s n,xhci,1-1:1-2:1-3:1-4:2-1:2-2:2-3:2-4 \``
+
+   3) UOS access USB keyboard and mouse.
+   4) Exit UOS.
+   5) SOS tries to access USB keyboard and mouse, and fails.
+
+   **Impact:** SOS cannot use USB keyboard and mouse in such case.
+
+   **Workaround:** Unplug and plug-in the USB keyboard and mouse after exiting UOS.
+
+-----
+
+:acrn-issue:`2753` - UOS cannot resume after suspend by pressing power key
+   UOS cannot resume after suspend by pressing power key
+
+   **Impact:** UOS may failed to resume after suspend by pressing the power key.
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`2974` - Launching Zephyr RTOS as a real-time UOS takes too long
+   Launching Zephyr RTOS as a real-time UOS takes too long
+
+   These steps reproduce the issue:
+
+   1) Build Zephyr image by follow the `guide
+      <https://projectacrn.github.io/latest/tutorials/using_zephyr_as_uos.html?highlight=zephyr>`_.
+   2) Copy the "Zephyr.img", "OVMF.fd" and "launch_zephyr.sh" to ISD.
+   3) execute the launch_zephyr.sh script.
+
+   **Impact:** Launching Zephyr RTOS as a real-time UOS takes too long
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`3268` - dm: add virtio-rnd device to command line
+   LaaG's network is unreachable with uos kernel
+
+   These steps reproduce the issue:
+
+   1) Downoad clearlinux
+   2) Decompress clearlinux
+   3) Replace above kvm.img with default kernel in uos
+   4) Launch UOS
+   5) Try to ping uos from other host.
+   6) UOS network is unreachalbe.
+
+   **Impact:** LaaG's network is unreachable with uos kernel
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`3281` - AcrnGT emulation thread causes high cpu usage when shadowing ppgtt
+   AcrnGT emulation thread causes high cpu usage when shadowing ppgtt
+
+   **Impact:** AcrnGT emulation thread causes high cpu usage when shadowing ppgtt
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`3280` - AcrnGT holding forcewake lock cause high cpu usage gvt workload thread.
+   AcrnGT holding forcewake lock cause high cpu usage gvt workload thread.
+
+   **Impact:** AcrnGT holding forcewake lock cause high cpu usage gvt workload thread.
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`3279` - AcrnGT causes display flicker in some situations.
+   AcrnGT holding forcewake lock cause high cpu usage gvt workload thread.
```

Comment:
This detail and impact are wrong (copied from the previous issue)

Reply:
Will fix it in next release